### PR TITLE
F 51 refactor event processing

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related #51

### CODE REVIEW

**Comment by mohammadranjbarz** [comment]
@aminlatifi  I had problem on installing dependencies, so I updated giveth bridge dependencies as well

**Review by aminlatifi — COMMENTED** [review]
Please update the event index to facilitate the sorting and finding the first event to process. Current index is
`{ isHomeEvent: 1, blockNumber: 1, transactionIndex: 1, logIndex: 1 },`

File: feathers-src/models/transactions.model.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+function Transaction(app) {
+  const mongooseClient = app.get('mongooseClient');
+  const { Schema } = mongooseClient;
+  const transaction = new Schema(
+    {
+      hash: { type: String, required: true, index: true },
+      from: { type: String },
+      gasPrice: { type: Number },
```
[the remark this answers is not in the document]
Comment (mohammadranjbarz) [reply feathers-src/models/transactions.model.js]: You are right, extra fields removed
@aminlatifi

File: feathers-src/lib/populate.js [hunk 2]
```diff
@@ -38,6 +38,9 @@ const processNextWaitingEvent = async () => {
   const query = {
     status: EventStatus.WAITING,
     $sort: {
+      // maybe we get some events from home and blockchain with different blockNumber, so the transactionTime
+      // should be the first important thing for decide what event should process first
+      timestamp: 1,
       isHomeEvent: 1,
```
[the remark this answers is not in the document]
Comment (mohammadranjbarz) [reply feathers-src/lib/populate.js]: `isHomeEvent` removed

**Pull request merged**